### DISCUSSION TITLE
Magic Key Words

### DISCUSSION THREAD

**Discussion by microraptor** [nerdyrodent/VQGAN-CLIP#39] [Show and tell]
Please share any good key words that work well with VQGAN+CLIP you know or resources for them here.

Some I heard so far:

- unreal engine | hyperrealistic | vray
- trending on artstation
- photorealistic
- render
- psychedelic | surreal | weird
- pencil art sketch
- drawn by a child
- in the style of xxx

**Comment by nerdyrodent** [comment]
There is a good list of examples at https://twitter.com/98_0634741763/status/1418440857578098691?s=21

**Comment by nerdyrodent** [comment]
"It aint what you say, it's the way that you say it" test. 
Format example:
1. a mushroom|academic art:0.5
2. a mushroom, academic art
3. a mushroom|academic art
4. a mushroom academic art

RMSProp:
![collage](https://user-images.githubusercontent.com/74688049/132142512-f9a0a5be-14e1-4ff3-a9ef-551ac16465e1.jpg)

DiffGrad:
![mushroom_collage](https://user-images.githubusercontent.com/74688049/132142902-19b77b62-6f39-4bee-a882-134e000d66a5.jpg)

**Reply by nerdyrodent** [reply]
![mushroom_collage](https://user-images.githubusercontent.com/74688049/132143505-60d1546f-4d4a-4083-896a-e03bb380d90d.jpg)

**Comment by microraptor** [comment]
There is also #13

**Comment by gateway** [comment]
great resource :) https://imgur.com/a/SALxbQm

**Comment by giantmonster** [comment]
Try these: https://en.wikipedia.org/wiki/List_of_art_movements
